README.md

```diff
@@ -1,4 +1,14 @@
-# The OpenJS Foundation Cross Project Council
+<p align="center">
```

Comment:
```suggestion
<p align="left">
```
it looks odd centered when the rest is left-aligned

Reply:
We don't need the `<p` statement at all then if it's all left aligned no? We can even switch to pure Markdown I guess...

Reply:
If we follow this idea we will no longer have the dark mode switch

Reply:
The image tag can still be HTML, but all the rest can be Markdown.